### ISSUE TITLE
Language selection is ignored during installation

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#4285]
Description: 
The Shopware 6 installer ignores the language setting during the installation. If you set the language completely to German, the administration will load in English.

Steps to reproduce: 
- Install Shopware 6: ```
composer create-project -q shopware/production
```
- Go through the installation process completely and select the German language

Expected result: 
- The administration / first-run wizard should be output in German.

Actual result: 
- The administration interface is output in English. Only after the personal user account has been switched to German, the administration interface will be displayed in German.

**Comment by shopwareBot** [comment]
[public] Automated response: This issue is linked to the internal issue [https://shopware.atlassian.net/browse/NEXT-32907](https://shopware.atlassian.net/browse/NEXT-32907).

 

[created from NEXT-32907, comment 486132]

**Comment by shopwareBot** [comment]
[public] Hello,

 Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

 If you feel like this issue is still impacting you, please create a new issue and let us know. Thank you again for your valuable feedback.



[created from NEXT-32907, comment 492308]
